src/test/java/core/basesyntax/dao/FileReaderImplTest.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.dao;
```

Comment:
why did you put FileReaderImplTest under dao package? is it something related to DAO? I think not